python/dgl/distributed/graph_partition_book.py

```diff
@@ -1363,26 +1414,32 @@ class HeteroDataName(object):
 
     def __init__(self, is_node, entity_type, data_name):
         self._policy = NODE_PART_POLICY if is_node else EDGE_PART_POLICY
+        if not is_node:
+            assert isinstance(entity_type, tuple) and len(entity_type) == 3, \
+                f"Expect canonical edge type in a triplet of string, but got {entity_type}."
```

Comment:
over 80 chars.

Reply:
fixed in https://github.com/dmlc/dgl/pull/4809